### DISCUSSION TITLE
Manual/Preferences

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#590] [Notes]
# Manual/Preferences

https://docs.unity3d.com/Manual/Preferences.html

**Comment by vertxxyz** [comment]
2023.3.0a3 adds `VisualElement.disablePlayModeTint` to opt-out of Play mode tinting. It applies hierarchically so you can apply it to [`EditorWindow.rootVisualElement`](https://docs.unity3d.com/ScriptReference/EditorWindow-rootVisualElement.html) to disable Play mode tinting on a whole window.

**Comment by andreasIsCoding** [comment]
You can check which editor theme is in use with `EditorGUIUtility.isProSkin()`.  You can set the current theme using the undocumented api `InternalEditorUtility.SwitchSkinAndRepaintAllViews ()`

**Comment by andreasIsCoding** [comment]
Note that if you have assemblies locked, toggling between light and dark theme in the preferences will appear to do nothing.  The theme will immediately change upon unlocking the assemblies.
